### ISSUE TITLE
creating then deleting a channel in a legacy project deletes all other tables in the legacy project

### ISSUE THREAD

**Issue by alexbaden** [openconnectome/open-connectome#167]
Maybe we can disable channel deletion in legacy projects until this is fixed? 

@kunallillaney @randalburns

**Comment by kunallillaney** [comment]
You should not be creating channels in legacy projects.

**Comment by alexbaden** [comment]
Yeah, I think we should try and find a way to prevent this in the admin interface. Either prevent creating, or prevent deleting. Easiest way seems to be to prevent deleting...
